### PR TITLE
Fixing bugs in regenerate

### PR DESCRIPTION
**Pull request by rzhao-2**
also deprecated dereplicator as type_strains_list is no longer needed

### CODE REVIEW

File: bin/singlem [hunk 1]
```diff
@@ -417,7 +417,7 @@ if __name__ == '__main__':
     required_regenerate_arguments.add_argument('--intermediate_archaea_graftm_package', required=True)
     required_regenerate_arguments.add_argument('--intermediate_bacteria_graftm_package', required=True)
     required_regenerate_arguments.add_argument('--input_taxonomy', required=True)
-    required_regenerate_arguments.add_argument('--type_strains_list_file', required=True)
+    # required_regenerate_arguments.add_argument('--type_strains_list_file', required=True)
```
Comment (wwood) [inline bin/singlem:420]: better to just delete this

File: bin/singlem [hunk 2]
```diff
@@ -792,8 +792,8 @@ if __name__ == '__main__':
             euk_taxonomy = args.euk_taxonomy,
             intermediate_archaea_graftm_package = args.intermediate_archaea_graftm_package,
             intermediate_bacteria_graftm_package = args.intermediate_bacteria_graftm_package,
-            input_taxonomy = args.input_taxonomy,
-            type_strains_list_file = args.type_strains_list_file)
+            input_taxonomy = args.input_taxonomy)
+            # type_strains_list_file = args.type_strains_list_file)
```
Comment (wwood) [inline bin/singlem:796]: delete

File: singlem/regenerator.py [hunk 3]
```diff
@@ -22,7 +22,7 @@ def regenerate(self, **kwargs):
         intermediate_archaea_graftm_package = kwargs.pop('intermediate_archaea_graftm_package')
         intermediate_bacteria_graftm_package = kwargs.pop('intermediate_bacteria_graftm_package')
         input_taxonomy = kwargs.pop('input_taxonomy')
-        type_strains_list_file = kwargs.pop('type_strains_list_file')
+        # type_strains_list_file = kwargs.pop('type_strains_list_file')
```
Comment (wwood) [inline singlem/regenerator.py:25]: delete

File: singlem/regenerator.py [hunk 4]
```diff
@@ -111,7 +111,7 @@ def regenerate(self, **kwargs):
             ' '.join(bacterial_intermediate_pkg.search_hmm_paths()),
             original_hmm_path,
             final_gpkg)
-        extern.run(cmd)
+        extern.run(cmd)
```
Comment (wwood) [inline singlem/regenerator.py:114]: please remove this trailing whitespace

File: singlem/regenerator.py [hunk 5]
```diff
@@ -55,12 +55,12 @@ def regenerate(self, **kwargs):
         num_euk_hits = 0
         final_sequences_path = os.path.join(working_directory,
                                             "%s_final_sequences.faa" % basename)
-        archeal_seqs = archaeal_intermediate_pkg.unaligned_sequence_database_path()
-        bacterial_seqs = bacterial_intermediate_pkg.unaligned_sequence_database_path()
-        with open(type_strains_list_file) as f:
-            type_strain_identifiers = [s.strip() for s in f.readlines()]
-        logging.info("Read in %i type strain IDs e.g. %s" % (
-            len(type_strain_identifiers), type_strain_identifiers[0]))
+        # archeal_seqs = archaeal_intermediate_pkg.unaligned_sequence_database_path()
```
Comment (wwood) [inline singlem/regenerator.py:58]: and delete - you get the idea - there's more below too

**Review by wwood — CHANGES_REQUESTED** [review]
hey just a few small things to fix. Thanks.

**Comment by rzhao-2** [comment]
should be good now, also adding a fix to seqs.

**Comment by wwood** [comment]
you already added a fix? I see that the new commits do that. Should I wait to merge?

**Comment by rzhao-2** [comment]
The seqs fix is already in there, it's all good to go.

**Comment by wwood** [comment]
rzhao-2: Fixes for regenerate and seqs.

**Comment by wwood** [comment]
Thanks

**Pull request merged**